### PR TITLE
qa/tests: reduced octopus-x jobs on master and pacific (--subset 1/10)

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by liewegas — APPROVED** [review]

File: qa/crontab/teuthology-cronjobs [hunk 1]
```diff
@@ -124,7 +124,7 @@ CEPH_QA_EMAIL="ceph-qa@ceph.io"
 17 05 * * 1,5       CEPH_BRANCH=pacific; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 100 -m $MACHINE_NAME -s krbd -k testing -e $CEPH_QA_EMAIL
 23 14 * * 3,6       CEPH_BRANCH=pacific; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -k distro -n 100 -m $MACHINE_NAME -s upgrade/nautilus-x -e $CEPH_QA_EMAIL
 20 01 * * 1,5       CEPH_BRANCH=pacific; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 100 -m $MACHINE_NAME -s upgrade-clients/client-upgrade-octopus-pacific -k distro -e $CEPH_QA_EMAIL --suite-branch octopus
-22 14 * * 2,6       CEPH_BRANCH=pacific; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c   $CEPH_BRANCH -k distro -n 100 -m $MACHINE_NAME -s upgrade/octopus-x -e $CEPH_QA_EMAIL -p 70 --force-priority
+22 14 * * 2,6       CEPH_BRANCH=pacific; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c   $CEPH_BRANCH -k distro -n 100 -m $MACHINE_NAME -s upgrade/octopus-x -e $CEPH_QA_EMAIL -p 70 --force-priority --subset 1/10
```
Comment (batrick) [inline qa/crontab/teuthology-cronjobs:127]:
```suggestion
22 14 * * 2,6       CEPH_BRANCH=pacific; MACHINE_NAME=smithi; SUITE_NAME=upgrade:octopus-x;      KERNEL=distro; /home/teuthology/bin/cron_wrapper /home/teuthology/bin/schedule_subset.sh 10 $CEPH_BRANCH $MACHINE_NAME $SUITE_NAME $CEPH_QA_EMAIL $KERNEL -p 70 --force-priority
```

File: qa/crontab/teuthology-cronjobs [hunk 2]
```diff
@@ -135,5 +135,5 @@ CEPH_QA_EMAIL="ceph-qa@ceph.io"
 
 #20 01 * * 4  CEPH_BRANCH=master; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 100 -m $MACHINE_NAME -s upgrade-clients/client-upgrade-octopus-pacific -k distro -e $CEPH_QA_EMAIL --suite-branch octopus
 
-#22 14 * * 5 CEPH_BRANCH=master; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c   $CEPH_BRANCH -k distro -n 100 -m $MACHINE_NAME -s upgrade/octopus-x -e $CEPH_QA_EMAIL
+#22 14 * * 5 CEPH_BRANCH=master; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c   $CEPH_BRANCH -k distro -n 100 -m $MACHINE_NAME -s upgrade/octopus-x -e $CEPH_QA_EMAIL --subset 1/10
```
Comment (batrick) [inline qa/crontab/teuthology-cronjobs:138]: ditto

**Review by batrick — CHANGES_REQUESTED** [review]
you can use `schedule-subset.sh`.

File: qa/crontab/teuthology-cronjobs (hunk 2 above)
Comment (batrick) [reply qa/crontab/teuthology-cronjobs:138]: yuri, this should also use schedule-subset.sh

Comment (yuriw) [reply qa/crontab/teuthology-cronjobs]: got it now

**Comment by yuriw** [comment]
wait a sec...

**Review by batrick — APPROVED** [review]

**Pull request merged**